modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Select a *Persistent storage* capacity and state the number of *Load balancers* that you require for your cluster.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. The option is enabled by default.
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. The option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+====
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type* and a *Compute node count*. The number and types of nodes that are available depend on your {product-title} subscription. If you are using multiple availability zones, the compute node count is per zone.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type. The number and types of nodes available to you depend on your {product-title} subscription.
+====
+
+. Optional: Expand *Edit node labels* to add labels to your nodes. Click *Add label* to add more node labels.
+
+. In the *Cluster privacy* dialog, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster.
+
+. In the *CIDR ranges* dialog, configure custom classless inter-domain routing (CIDR) ranges or use the defaults that are provided.
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+
+If the cluster privacy is set to *Private*, you will not be able to access your cluster until you configure private connections in your cloud provider.
+====
+
+. On the *Cluster update strategy* page, configure your update preferences:
+.. Choose a cluster update method. By selecting *Individual updates*, you can schedule each update individually. If you select *Recurring updates*, your cluster is updated on your preferred day and start time, when updates are available. *Individual updates* is the default option.
++
+[NOTE]
+====
+You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
```

Comment:
Same comment about life cycle vs. lifecycle

Reply:
I will leave this as it is for now, for the reasons I outlined in the preceding comment thread.